sope-mime/NGMail/NGMailAddressParser.m

```diff
@@ -406,7 +406,8 @@ + (NSString *) sanitizeMailAddresses:(NSString *)originalAddress
     r = [remainder rangeOfString: @"<"];
     addrStart = r.location;
 
-    c = [remainder rangeOfString: @","];
+    a = [remainder rangeOfString: @"@"];
```

Comment:
And if we call this variable 'rangeOfAtCharacter' it is tricky because the character is call 'at'